plugins/services/src/js/components/modals/ServiceModals.js

```diff
@@ -7,6 +7,7 @@ import ServiceTree from '../../structs/ServiceTree';
 import ServiceActionItem from '../../constants/ServiceActionItem';
 import ServiceDestroyModal from './ServiceDestroyModal';
 import NewCreateServiceModal from './NewCreateServiceModal';
+import NewApplicationDefaults from '../../constants/NewApplicationDefaults';
```

Comment:
Check my comment above, but perhaps it makes sense to import this as `NEW_APP_DEFAULTS` ?